siliconcompiler/package.py

```diff
@@ -44,7 +45,13 @@ def path(chip, package, quiet=True):
         return path
 
     # location of the python package
-    cache_path = os.path.join(Path.home(), '.sc', 'cache')
+    cache_path = chip.get('option', 'cache')
+    if cache_path:
+        cache_path = chip.find_files('option', 'cache', missing_ok=True)
+        if not cache_path:
```

Comment:
So can cache path can be both a relative and absolute path?

Reply:
yes don't see why not.